src/main/java/org/dita/dost/reader/CopyToReader.java

```diff
@@ -86,12 +87,9 @@ public void startElement(final String uri, final String localName, final String
         processRoleStack.push(processingRole);
 
         final String classValue = atts.getValue(ATTRIBUTE_NAME_CLASS);
-
-        if (foreignLevel > 0) {
-            foreignLevel++;
-        } else if (TOPIC_FOREIGN.matches(classValue) || TOPIC_UNKNOWN.matches(classValue)) {
-            foreignLevel++;
-        }
+        
+        final DitaClass cls = atts.getValue(ATTRIBUTE_NAME_CLASS) != null ? new DitaClass(atts.getValue(ATTRIBUTE_NAME_CLASS)) : null;
```

Comment:
Want to make sure I understand it correctly ... current code is 
```
 final DitaClass cls = atts.getValue(ATTRIBUTE_NAME_CLASS) != null ? new DitaClass(atts.getValue(ATTRIBUTE_NAME_CLASS)) : null;
 classes.addFirst(cls);
```
So -- if the attribute `@class` exists, create a new `DitaClass` and add to the stack (allowing tests for validity as needed), otherwise add `null` ... is the idea that if the class isn't DITA in the first place, we should add `null`? That sounds logical, just wanted to double check